cognite/client/_api/files.py

```diff
@@ -527,6 +553,7 @@ def upload(
             asset_ids=asset_ids,
             data_set_id=data_set_id,
             labels=labels,
+            geo_location=geo_location,
```

Comment:
Similar suggestion as first one. If there's any specific reason then I guess it's fine here as well.